src/storage/hummock_test/src/test_utils.rs

```diff
@@ -18,7 +18,9 @@ use std::future::Future;
 use std::ops::Deref;
 use std::sync::Arc;
 
+use async_stream::try_stream;
```

Comment:
`async_stream` is built with channels and performs worse than `futures_async_stream`. I guess we should prefer the latter.

Reply:
I see. I will adopt it.

Reply:
I have switched to use `futures_async_stream` in `risingwave_storage`. The `hummock_test` crates is still using `async_stream` for simplicity.